### PR TITLE
Provide helm chart for Eclipse Ditto

### PR DESCRIPTION
**Pull request by ghandim**
Signed-off-by: Michael Gantert <michael.gantert@bosch-si.com>

### CODE REVIEW

**Comment by kaizimmerm** [comment]
Hi, 

I can confirm that this works on AKS as well. 

As soon as you have this on master I can offer to contribute a few more flexibility options, i.e. service type, namespace support etc.

Kai

**Comment by kaizimmerm** [comment]
@thjaeckle do you have already an idea when this will be on master?

**Comment by Yannic92** [comment]
It's currently in review and some changes are required. Can't give you an exact date.

File: deployment/helm/eclipse-ditto/files/openapi/ditto-api-1.yml [hunk 1]
```diff
@@ -0,0 +1,3060 @@
+# Copyright (c) 2017-2018 Bosch Software Innovations GmbH.
```
Comment (Yannic92) [inline deployment/helm/eclipse-ditto/files/openapi/ditto-api-1.yml]: I think we shouldn't duplicate the swagger documentation. How about a symbolic link?

File: deployment/helm/eclipse-ditto/files/openapi/ditto-api-2.yml [hunk 2]
```diff
@@ -0,0 +1,4134 @@
+# Copyright (c) 2017-2018 Bosch Software Innovations GmbH.
```
Comment (Yannic92) [inline deployment/helm/eclipse-ditto/files/openapi/ditto-api-2.yml]: I think we shouldn't duplicate the swagger documentation. How about a symbolic link?

**Comment by kaizimmerm** [comment]
@Yannic92 great! I have actually quite a few changes that I can PR as soon as you have this on master. So far:

- Support for namespace deployment.
- Support for other service types then `NodePort`.
- Support for external MongoDB including SSL support.
- Added [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/).
- Support for other Docker repositories or tags.

File: deployment/helm/eclipse-ditto/files/openapi/ditto-api-1.yml (hunk 1 above)
Comment (ghandim) [reply deployment/helm/eclipse-ditto/files/openapi/ditto-api-1.yml]: Thanks for the hint. Prepare an additional commit on this PR.

File: deployment/helm/eclipse-ditto/files/openapi/ditto-api-2.yml (hunk 2 above)
Comment (ghandim) [reply deployment/helm/eclipse-ditto/files/openapi/ditto-api-2.yml]: Thanks for the hint. Prepare an additional commit on this PR.

**Pull request merged**